### PR TITLE
Extract and document json schema

### PR DESCRIPTION
**Pull request by dethe**
Fixes https://github.com/participedia/api/issues/7

### CODE REVIEW

File: json_stats.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import json, sys, re, random
+from bs4 import BeautifulSoup
+
+DATE_FMT = re.compile(r'\d{4}-\d{2}-\d{2}')
+INT_FMT = re.compile(r'^\d+$')
+FLOAT_FMT = re.compile(r'^\d+\.\d+$')
+WHITESPACE_FMT = re.compile(r'\s+')
+
+def uprint(text=''):
+    print text.encode('utf-8')
```
Comment (davidascher) [inline json_stats.py]: hard to believe this is the right way to do things, but i'm waaay out of date on python + unicode.

**Comment by dethe** [comment]
Man, you're quick to merge! I was going to keep working on this PR and ping you when it was ready to merge. I like keeping a PR open while I work so I get Github's diff view, guess I should have said that.

**Comment by davidascher** [comment]
Sorry!

re: encoding UTF8 - IIRC, the way I used to do it was to specify the
encoding used by sys.stdout, and then just print normally.

```
import sys
import codecs
sys.stdout = codecs.getwriter('utf8')(sys.stdout)
```

but whatever.

**Comment by dethe** [comment]
I saw that as an option too, but everytime someone mentioned it, others chimed in warning that you could break all kinds of things that depend on stdout. I rationally know that it is extremely unlikely to break anything for our use case, but I still went with what I was sure would work and not break things.

**Comment by davidascher** [comment]
yup, all good.

**Pull request merged**